docs/onboarding/installation.md

```diff
@@ -31,23 +28,11 @@ ironfish
 
 <br />
 
-When updates are available, update Brew, then reinstall Iron Fish.
-
-```sh
-brew update && brew uninstall ironfish && brew install ironfish
-```
-
 Iron Fish is now ready to use. Follow the [next step](new_node.md) of the tutorial or jump directly to the [the CLI commands list](cli.md).
 
-## Windows
-
-Windows installer packages are still in development. In the meantime, you can install Iron Fish directly [from the source](#from-source) or [use Docker](#using-docker).
-
-## Linux
-
-Linux installer packages will be published soon. In the meantime, you can install Iron Fish directly [from the source](#from-source) or [use Docker](#using-docker).
-
-## Using Docker
+## Alternative Installation Methods
+You can also get started with Iron Fish by using Docker or running from source. If you are using an Intel-based macOS system, we support Homebrew as an alternative installation method.
+### Using Docker
```

Comment:
Minor (and maybe completely unhelpful), but does this headline need a preceding empty line?

Reply:
Sure does!